modules/cert-manager-certificate-ingress.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-creating-certificate.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-certificate-ingress_{context}"]
+= Creating certificates for Ingress Controller
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed {cert-manager-operator} 1.13.0 or above.
+
+.Procedure
+
+. Create an issuer. For more information, see "Configuring an issuer" in "Additional Resources" section.
+
+. Create a certificate:
+
+.. Create a YAML file, for example, `certificate.yaml`, that defines the `Certificate` object:
++
+.Example `certificate.yaml` file
++
+[source, yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: <tls_cert> #<1>
+  namespace: openshift-ingress
+spec:
+  isCA: false
+  commonName: "<common_name>" #<2>
+  secretName: <tls_cert> #<3>
+  dnsNames:
+  - "*.new-ingress.<sub.domai.ns-domain-x.y.z.com>" #<4>
```

Comment:
This has to be:
```
dnsNames:
- apps.<cluster_domain_name>
- *.apps.<cluster_domain_name>
```
and commonName field has to be exact same as the first one i.e.
```
commonName: apps.<cluster_domain_name>
```

Reply:
do you think the corresponding callouts need any changes or does the existing description work?